package.json

```diff
@@ -127,7 +127,7 @@
     "@bahmutov/cypress-esbuild-preprocessor": "^2.2.0",
     "@cypress/code-coverage": "^3.11.0",
     "@department-of-veterans-affairs/eslint-plugin": "^1.18.2",
-    "@department-of-veterans-affairs/generator-vets-website": "^3.9.2",
+    "@department-of-veterans-affairs/generator-vets-website": "^3.10.0",
```

Comment:
Make sure to run `yarn` locally and commits the `yarn.lock` file updates too.

Reply:
@jamigibbs  hm, vets-website won't recognize the new version of @department-of-veterans-affairs/generator-vets-website when doing yarn, it still only sees 3.9.2 as the latest. do we have to do something so generator-vets-website is so 3.10.0 is publicly viewable by vets-website, or publish or something?

Reply:
nevermind. All good now. it needed to be published to npm.